docs/.vuepress/config.js

```diff
@@ -73,6 +73,98 @@ const sidebar = [
     children: [['/automating/', 'Getting started']],
   },
 ];
+
+const sidebar = [
+  ['/', 'Home'],
+  ['/guide/', 'Introduction'],
+  {
+    title: 'Guides & Docs',
+    collapsable: true,
+    children: [
+      '/developing/best-practices',
+      ['/codelabs/', 'Codelabs'],
+      '/developing/code-examples',
+      '/developing/lit-html',
+      '/developing/types',
```

Comment:
Remove types, for now, => it's not up to date anymore... we can improve and readd later again 👍